BUILD.bazel

```diff
@@ -0,0 +1,212 @@
+load("//bazel:swift_cc_defs.bzl", "UNIT", "swift_cc_library", "swift_cc_test")
+
+swift_cc_library(
```

Comment:
Make sure that we also set the warning flags for this project: https://github.com/swift-nav/albatross/blob/01e266027fa4096165966f1b6b847b4f11944a2d/CMakeLists.txt#L45

Reply:
Since this target contains only headers, I added warning flags to the `albatorss-test` target and examples binary.